### ISSUE TITLE
Implement optional 'penalty' for svr regression

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/machine-learning#3131]
We will expose the `C` penalty parameter, for both the web-interface, and programmatic rest-api, with respect to the [`svr`](http://scikit-learn.org/stable/modules/generated/sklearn.svm.SVR.html) regression.

**Comment by jeff1evesque** [comment]
We should discuss the significance of the `C` parameter, with respect to `svr`, in our `doc/`.

**Comment by jeff1evesque** [comment]
The following reactjs component could be implemented for the frontend:

```javascript
import React, { Component } from 'react'
import Slider from 'react-rangeslider'

class Float extends Component {
  constructor (props, context) {
    super(props, context)
    this.state = {
      value: 12.5
    }
  }

  handleChange = (value) => {
    this.setState({
      value: value
    })
  }

  render () {
    const { value } = this.state
    return (
      <div className='slider'>
        <Slider
          min={10}
          max={20}
          step={0.5}
          value={value}
          onChange={this.handleChange}
        />
        <div className='value'>{value}</div>
      </div>
    )
  }
}

export default Float
```

**Note:** the corresponding example can be reviewed:

- https://whoisandy.github.io/react-rangeslider/

**Comment by jeff1evesque** [comment]
This issue has been resolved via #3132. However, since the associated documentation has some typos, we'll use this issue, to make corresponding fixes.
